improver/cli/estimate_emos_coefficients.py

```diff
@@ -88,12 +92,13 @@ def process(
             coefficient is stored in a separate cube.
     """
 
-    from improver.calibration import split_forecasts_and_truth
     from improver.calibration.ensemble_calibration import (
         EstimateCoefficientsForEnsembleCalibration,
     )
+    from improver.calibration import get_cube_from_directory
 
-    forecast, truth, land_sea_mask = split_forecasts_and_truth(cubes, truth_attribute)
+    forecast = get_cube_from_directory(forecast_directory, cycle_point=cycle_point, max_days_offset=max_days_offset)
+    truth = get_cube_from_directory(truth_directory, cycle_point=cycle_point, max_days_offset=max_days_offset)
```

Comment:
I appreciate this is still in draft so likely you are doing a minimal change for testing.  However, if/when you come closer to pulling this out of draft, we would want to ensure feature parity between the CLI and the plugin, whetehr that means changing `EstimateCoefficientsForEnsembleCalibration` UI and behaviour to call `get_cube_from_directory`, or otherwise defining a metaplugin to do achieve this.

Reply:
I had thought about this issue, but hadn't considered a metaplugin, which I think might be the way to go. Thanks :)